### PR TITLE
Run world latest

### PR DESCRIPTION
**Pull request by davide-f**
# Closes # (if applicable).

## Changes proposed in this Pull Request
Include fixes and improvements to enable the execution of the default configuration for every country in the world

## Checklist

- [ ] I consent to the release of this PR's code under the AGPLv3 license and non-code contributions under CC0-1.0 and CC-BY-4.0.
- [ ] I tested my contribution locally and it seems to work fine.
- [ ] Code and workflow changes are sufficiently documented.
- [ ] Newly introduced dependencies are added to `envs/environment.yaml` and `doc/requirements.txt`.
- [ ] Changes in configuration options are added in all of `config.default.yaml` and `config.tutorial.yaml`.
- [ ] Add a test config or line additions to `test/` (note tests are changing the config.tutorial.yaml)
- [ ] Changes in configuration options are also documented in `doc/configtables/*.csv` and line references are adjusted in `doc/configuration.rst` and `doc/tutorial.rst`.
- [ ] A note for the release notes `doc/release_notes.rst` is amended in the format of previous release notes, including reference to the requested PR.

### CODE REVIEW

File: scripts/simplify_network.py [hunk 1]
```diff
@@ -161,12 +161,13 @@ def _prepare_connection_costs_per_link(n, costs, config):
...
-        dc_lengths = n.links.length
-        unterwater_fractions = n.links.underwater_fraction
-    elif not n.lines.loc[n.lines.carrier == "DC"].empty:
+    # initialize dc_lengths and underwater_fractions by the hvdc_as_lines option
+    if config["electricity"]["hvdc_as_lines"]:
         dc_lengths = n.lines.length
         unterwater_fractions = n.lines.underwater_fraction
+    else:
+        dc_lengths = n.links.length
+        unterwater_fractions = n.links.underwater_fraction
```
Comment (ekatef) [inline scripts/simplify_network.py:170]: Great fix! I think, it looks cleaner now :)

As a side comment, `dc_length` naming looks quite misleading here... Would you mind to remove "dc_" prefix from it?

**Review by ekatef — APPROVED** [review]

**Comment by ekatef** [comment]
Hello Davide! :D

My impression is that you have managed to make the code cleaner when adding the fixes. Fantastic work! 

Basically, I have not much to add. The amended suggestion relates to the old version, not to the PR itself. Feel free to disregard in case 🙂

File: scripts/simplify_network.py (hunk 1 above)
Comment (davide-f) [reply scripts/simplify_network.py:170]: Actually, the reason of these lines it to capture the lines that represent DC lines, be them represented as lines or links.
I think the naming should be ok.
Do you think it needs revision?

Comment (ekatef) [reply scripts/simplify_network.py:170]: > Actually, the reason of these lines it to capture the lines that represent DC lines, be them represented as lines or links.
> 
> I think the naming should be ok.
> 
> Do you think it needs revision?

I needed some time to remember what does dc_lenghs mean, when looking into the code :) But naming is generally tricky, and if you feel it's alright then it most likely is :)

**Pull request merged**